### DISCUSSION TITLE
问一些关于mpcvr的问题

### DISCUSSION THREAD

**Discussion by TokuSeiR** [hooke007/mpv_PlayKit#233] [其它播放器]
大佬，我这边有两台笔记本设备，显卡方面一台使用英伟达的2070MQ，一台使用截图里面的英特尔核显，在我这边在使用mpcvr的时候，在控制面板，视频渲染器都开启0-255，视频源也是0-255的前提下，mpcvr的颜色范围会被限制在16-235，我无论怎么更改设置都达不到大佬你的mpcvr汉化项目里面的0-255，画面对比madvr和mpv更是明显发灰，请问该如何解决呢？
![屏幕截图(1)](https://user-images.githubusercontent.com/45761551/203241772-8a330ef3-2bba-4856-85fb-48e590ca8c6f.png)

**Comment by hooke007** [comment]
那个项目有声明去哪里反馈问题

**Reply by hooke007** [reply]
首先尝试着色器渲染器

**Reply by TokuSeiR** [reply]
因为现在在上课，所以目前手里只有那台设备，核显设备开启EVR也是16-235，所以是片源问题还是笔记本平台的问题呢

**Reply by hooke007** [reply]
我说的是用着色器渲染器（禁用d3d11视频渲染器），这些词的解释在主页有说明
没说evr

**Reply by TokuSeiR** [reply]
![屏幕截图(2)](https://user-images.githubusercontent.com/45761551/203245934-5bd37de5-2eb7-4f73-9312-ad23780dfbc5.png)
关了d3d11

**Reply by hooke007** [reply]
你的解码器设置界面是什么样子

**Reply by TokuSeiR** [reply]
关了d3d11还是不行

**Reply by TokuSeiR** [reply]
![屏幕截图(3)](https://user-images.githubusercontent.com/45761551/203247615-01e9533d-e154-4e2f-a7a6-fe6ef14bebdb.png)

**Reply by hooke007** [reply]
> 视频源也是0-255的前提下

你这片源是16-235

**Reply by hooke007** [reply]
input format 是输入格式，显示0-255是因为我的测试片源是全范围

如果有颜色差异，在**使用d3d11**(和使用**d3d11视频处理器**是两个东西)和**着色器视频处理器**的前提下和mpv截屏对比

**Reply by TokuSeiR** [reply]
但是对比之下，madvr确实可以呈现0-255，这也是我如此纠结这个问题的其中一个原因

**Reply by hooke007** [reply]
我没有根据苍白的文字来脑补的超能力。截图说明

**Reply by TokuSeiR** [reply]
那么请问大佬，哪里可以找到0-255的测试片源呢

**Reply by hooke007** [reply]
你主楼的图的白点已经达到250了，说明根本不是输出235
顺带一提 https://github.com/hooke007/MPV_lazy/discussions/233#discussioncomment-4201989 这图黑点 0 ，输出全范围无误。这个场景也是典型的可对比场景，两个播放器播放这一帧如果有颜色差异一眼就能看出来

**Reply by hooke007** [reply]
没有必要测全范围的片源，黑白场景截图测黑白点

**Reply by TokuSeiR** [reply]
我也想确认是哪里出现了问题，要是片源问题，我就得去检查我主力机的madvr和mpv了

**Reply by TokuSeiR** [reply]
好的，谢谢大佬解答

**Reply by TokuSeiR** [reply]
辛苦大佬了
